common/src/main/java/com/revenuecat/purchases/common/verification/SigningManager.kt

```diff
@@ -36,6 +39,10 @@ class SigningManager(
         requestTime: String?,
         eTag: String?,
     ): VerificationResult {
+        if (appConfig.forceSigningErrors) {
+            warnLog("Forcing signing error for request with path: $urlPath")
+            return VerificationResult.FAILED
+        }
```

Comment:
I wish there was a way to ensure this cannot run in production builds

Reply:
Yeah honestly it's not great... With the server version of this I was thinking of a way by creating a new class that exists in each flavor (the integration test and the defaults flavors). The defaults flavor version wouldn't do anything and the integration test flavor would indicate the failure, however, we would still have to use that code here and it would complicate things more... I would really like to have what we have in iOS where we can make code compile only with certain flags but that's more difficult to architecture in Android and not sure if it's worth it...

Reply:
Good idea! We actually can get the flavor from `BuildConfig.FLAVOR` without extra files. We can use that to do what you said yeah. It would still have this code in the final bundle, but it would be another safety measure, I can do it like that 👍

Reply:
Maybe you're thinking of something better, but we could change `AppConfig` so that some values can never return anything but the default depending on `BuildConfig.FLAVOR`.

Reply:
Done in the base PR: https://github.com/RevenueCat/purchases-android/pull/1070/commits/e16c4107173901a22915e6d16088a820c14e3415